library/data/registry-item-attributes/src/main/java/org/quiltmc/qsl/registry/attribute/api/package-info.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * <h2>Registry Item Attributes</h2>
+ *
+ * A simple API for adding arbitrary values to any object instances managed by a {@link net.minecraft.util.registry.Registry Registry}.<p>
+ *
+ * {@link org.quiltmc.qsl.registry.attribute.api.RegistryItemAttribute RegistryItemAttributes} contains a number of methods for creating
```

Comment:
Why does this rename `RegistryItemAttribute` to `RegistryItemAttributes`?

Reply:
That's an error, oops.